train.py

```diff
@@ -69,14 +75,33 @@ def main(args):
     Multi-frame query:
     ```shell
     python3 train.py \
-      config/regression/mapfree/multiframe/3d3d_multi.yaml config/mapfree.yaml
+      config/regression/mapfree/multiframe/3d3d_multi.yaml \
+      config/mapfree.yaml \
+      config/mapfree_multi.yaml
     ```
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument('config', help='path to config file')
-    parser.add_argument('dataset_config', help='path to dataset config file')
+    parser.add_argument('config', action='append', help='path to config file')
+    parser.add_argument('dataset_config', nargs='+', action='store',
+                        help='path to dataset config file')
+    parser.add_argument('--config', action='append', dest='config',
+                        help='one more path to a config file')
+    parser.add_argument('--dataset-config', '--dataset_config', action='append',
```

Comment:
Ok, so instead of a flag, we have to give an extra config file. I think this is very nice. 

One comment, if we want to have this logic, we should also implement it in the test script.

Reply:
It's not instead, it's in addition. So no changes needed I think.